hasher-matcher-actioner/terraform/main.tf

```diff
@@ -285,7 +285,11 @@ module "actions" {
   log_retention_in_days = var.log_retention_in_days
   additional_tags       = merge(var.additional_tags, local.common_tags)
   measure_performance   = var.measure_performance
-  te_api_token_secret      = aws_secretsmanager_secret.te_api_token
+  te_api_token_secret   = aws_secretsmanager_secret.te_api_token
+  config_table = {
+    name = aws_dynamodb_table.hma_config.name
+    arn  = aws_dynamodb_table.hma_config.arn
+  }
```

Comment:
nit: config_table = local.hma_config should work here, or you should be to make changes to the variable to make it work like fetcher does

Reply:
refactored and made consistent.